style.css

```diff
@@ -0,0 +1,452 @@
+    background-size: cover;
+    background-position: center;
+}
+
+.boxes{
+    overflow: auto;
+}
+
+/*ABOUT ME*/
+
+.aboutMe{
+    text-align: center;
+    overflow: auto;
+}
+
+.aboutMe h2{
+    font-size: 48px;
+    color: #565254;
+    font-weight: normal;
+    letter-spacing: 6px;
+}
+
+.aboutMe p{
+    letter-spacing: 2px;
+    line-height: 18px;
+    color: #565254;
+}
+
+.aboutMe .archWorks{
+    font-size: 24px;
+}
+
+.archBox{
+    width: 33.3%;
+    height: 250px;
+    float: left;
+    border: 1px solid black;
+}
+
+/*CONTACT ME*/
+
+footer{
+    text-align: center;
+    background-color: #272727;
+    width: 100%;
+    height: 160px;
+}
+
+footer h2{
+    margin-bottom: 10px;
+}
+
+footer p{
+    margin: 10px;
+}
+
+footer .contact{
+    padding-top: 5px;
+    color: #747474;
+}
+
+footer a{
+    text-decoration: none;
+    color: #FFFFFF;
+    opacity: 0.8;
+}
+footer .backUp{
+    padding: 5px 15px;
+    border: 1px solid transparent;
+    font-size: 16px;
+}
+
+footer .backUp:hover{
+    border: 1px solid #FFFFFF;
+}
+
+footer a:visited{
+    color: #FFFFFF;
+}
+
+.logoContainer a img{
+    height: 50px;
+    opacity: 0.8;
+    margin-left: 20px;
+}
+
+.logoContainer{
+    float: right;
+    display: inline-block;
+    margin-top: -45px;
+}
+
+.copyright{
+    color: #FFFFFF;
+    float: left;
+    margin-top: -15px;
+    font-weight: lighter;
+    opacity: 0.8;
+}
+
+/*MEDIA QUERIES*/
+
+@media only screen and (max-width:500px) {
+    .boxes .projectBox{
+        width: 100%;
+        height: 300px;
+    }
+    .navBar li{
+        font-size: 14px;
+        padding: 0;
+        margin: 30px 10px;
+        border: none;
+    }
+
+    .portfolio .projectBox h4{
+        margin-top: 20%;
+    }
+
+    .portfolio .projectBox h3{
+        margin-top: 20%;
+    }
+
+    .projectBox P{
+        font-size: 24px;
+        letter-spacing: 2px;
+        font-weight: bold;
+        color: #FFFFFF;
+        display: none;
+    }
+
+   footer{
+       height: 300px}
+}
+
+@media only screen and (max-width:768px) {
+    .navBar li a{
+        font-size: 14px;
+        margin: 0 20px;
+        padding: 10px;
+    }
+
+    .navBar ul{
+        margin: 0;
+    }
+
+    .navBar .navDefault{
+        display: none;
+    }
+    .navBar .navHidden{
+        display: block;
+    }
+
+    .featureImage h1{
+        letter-spacing: 20px;
+        line-height: 3.5rem;
+        font-size: 46px;
+    }
+
+    .featureImage p{
+        letter-spacing: 12px;
+        font-size: 28px;
+    }
+
+
+    .projectBox h3{
+        font-size: 32px;
+        letter-spacing: 3px;
+        margin-top: 30%;
+    }
+
+    .projectBox h4{
+        font-size: 32px;
+        letter-spacing: 3px;
+        margin-top: 30%;
+    }
+
+    .projectBox p{
+        font-size: 18px;
+        font-weight: normal;
+        margin-top: 10px;
+    }
+
+    footer{
+        height: 300px}
+}
+
+@media only screen and (min-width:768px) {
```

Comment:
You could have just changed the size of the actual class.  This media query is not needed